styles.css

```diff
@@ -38,28 +51,156 @@ body, html, a {
     list-style-type: none;
 }
 
-a {
-    color: #293241;
-}
-
-a:hover{
-    font-weight: 700;
-    cursor: pointer;
-}
-
-.sign-in:hover {
+.button:hover {
     background-color: #8b5d5d;
     border: 1px solid #000000;
     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
     cursor: pointer;
 }
 
-.sign-in {
+.button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
     height: 43px;
-    width: 122px;
+    width: 150px;
     background-color: #EE6C4D;
     border-radius: 48px;
+    border: none;
     font-weight: 700;
+    /* HERO section */
     font-size: 18px;
     color: #FFFFFF;
+}
+
+.section {
+    display: flex;
+    justify-content: center;
+    height: 812px;
+}
+
+.title {
+    margin-top: 160px;
+}
+
+.title h1 {
+    display: flex;
+    color: #EE6C4D;
+    width: 494px;
+    height: 225px;
+    font-size: 50px;
+    margin-top: 0px;
+    margin-bottom: 6px;
+}
+
+.presentation {
+    display: flex;
+    flex-direction: column;
+    margin: 100px 20px;
+    width: 1234px;
+    height: 631px;
+    background-image: url(images/hero.png);
+    background-position: right 60%;
+    background-repeat: no-repeat;
+}
+
+.description {
+    display: flex;
+    flex-direction: column;
+    width: 360px;
+    height: 90px;
+    margin-bottom: 24px;
+}
+
+@media (max-width: 1172px) {
+
+    .title h1 {
+        background-color: #FFFFFF;
+        border-radius: 20px;
+        box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.6), 0 10px 40px 0 rgba(0, 0, 0, 0.19);
+        padding: 6px;
+    }
+
+    .description {
```

Comment:
čia visiškai toks pats stilius kaip ir `.title h1`, todėl galima nekartoti to naties. Per kablelį išvardinti abu selectorius.